libgambatte/libretro/libretro_core_options.h

```diff
@@ -230,7 +230,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
          { "TWB64 078 - Christmas Ver.",            NULL },
          { "TWB64 079 - Cardcaptor Pink",           NULL },
          { "TWB64 080 - Pretty Guardian Gold",      NULL },
-         { "TWB64 081 - Camoflauge Ver.",           NULL },
+         { "TWB64 081 - Camouflage Ver.",           NULL },
```

Comment:
Ah, I just noticed this PR, and there are a couple of minor issues:

Changing this string will break palette selection unless it's also updated here: https://github.com/libretro/gambatte-libretro/blob/4b822bcf561856689e101dd5505d365de5ffcffe/libgambatte/libretro/gbcpalettes.h#L1921

Reply:
Oh, okay. Thanks for the heads up!